openmc/deplete/abc.py

```diff
@@ -833,6 +833,46 @@ def integrate(self, final_step=True, output=True):
 
         self.operator.finalize()
 
+    def plot(self):
+        """Plots the source strength as a function of time and the depletion
+        timesteps on an adjacent subplot.
+
+        Returns
+        -------
+        matplotlib.figure.Figure
+            Resulting image
+
+        """
+
+        import matplotlib.pyplot as plt
+
+        current_time = 0
+        linear_time_steps = [0]
+
+        for time_step in self.timesteps:
+            current_time += _get_time_as(time_step, self.timestep_units)
+            linear_time_steps.append(current_time)
+
+        fig, axes = plt.subplots(2, 1,  gridspec_kw={'height_ratios': [3, 1]})
+
+        # adds space between the plots to avoid overlapping x label
+        fig.subplots_adjust(hspace=.4)
+
+        axes[0].set_ylabel('Neutron source rate [n/s]')
+        axes[0].set_xlabel(f'Time [{self.timestep_units}]')
+        axes[0].stairs(self.source_rates, linear_time_steps, linewidth=2)
+
+        for timestep in linear_time_steps:
+            x_vals = [timestep, timestep]
+            y_vals = [0, 1]  # arbitrary heights selected as axis has no y scale
+            axes[1].plot(x_vals, y_vals, '-', color='red')
+
+        axes[1].set_xlabel(f'Timesteps [{self.timestep_units}]')
+        axes[1].set_ylim(0, 1)
+        axes[1].set_xticks(linear_time_steps)
```

Comment:
```suggestion
```

Sometimes the numbers can overlap so perhaps best to remove this custom x label

This plot for example would have overlapping x values, if this line is left in
 ![waste-compressed](https://user-images.githubusercontent.com/8583900/186134058-5b288b30-bb01-4609-a9a9-deea3381518c.png)

Reply:
done in commit #e0dcf17